### PR TITLE
Update README.md with JUnitParams

### PR DESCRIPTION
**Pull request by Cs4r**
Please, add JUnitParams to your README.md.

JUnitParams is a library which allows to create readable and maintainable parametrised tests. It is not widely known in the Java world, even though it provides much better functionalities than JUnit Parametrised runner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -534,6 +534,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
+* [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Parameterised tests that don't suck
```
Comment (akullpp) [inline README.md]: You forgot the dot at the end of the sentence.

Could you please add something more descriptive than "don't suck".

Comment (Cs4r) [inline README.md]: Ok. Done.

What do you think now?

**Comment by Cs4r** [comment]
Commit updated.

File: README.md [hunk 2]
```diff
@@ -534,6 +534,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
+* [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Creation of readable and maintainable parametrised tests. It provides much better functionalities than JUnit Parametrised runner.
```
Comment (akullpp) [inline README.md]: I'm sorry to be pedantic but if you make a claim "to be much better than x" you have to back it up with arguments. I would remove the second sentence.

Comment (Cs4r) [inline README.md]: Yes, you are. Although it's your repo and your rules prevail.

The latest commit contains your suggestions.

File: README.md [hunk 3]
```diff
@@ -534,6 +534,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hamcrest](http://hamcrest.org/JavaHamcrest/) - Matchers that can be combined to create flexible expressions of intent.
 * [JMockit](http://jmockit.org/) - Mocks static, final methods and more.
 * [JUnit](http://junit.org/) - Common testing framework.
+* [JUnitParams](https://pragmatists.github.io/JUnitParams/) - Creation of readable and maintainable parametrised tests.
```
Comment (akullpp) [inline README.md]: Spelling error: parametrised/parametrised

Comment (Cs4r) [inline README.md]: I do not see your point. What do you propose instead?

Comment (tedyoung) [inline README.md]: Looks like "parametrised" is the British English form of the past tense of parameterise. If we're sticking with American English spelling, then this should be "parameterized".

Comment (Cs4r) [inline README.md]: But both of them are correct. Don't they?

Comment (akullpp) [inline README.md]: Sorry, just took a quick glance. You are right, it is acceptable. No issue using BE or AE.

**Pull request merged**